custom_components/loxone/manifest.json

```diff
@@ -13,5 +13,5 @@
     "numpy",
     "httpx"
   ],
-  "version": "0.6.9"
+  "version": "0.6.10"
```

Comment:
Change in the `loxone` folder are intended? I think we shroud not change the application code. 

There may be permanent changes in the main project. If we ever merge back into the main project, there will only be conflicts and we have no intention of changing anything in the application code.

Reply:
Those changes aren't mine but directly form the main project.
I added them because I did no know what the button "Sync fork" does.
When thinking about it, it becomes obvious...

To get back to the matter: As those changes were created by @JoDehli I don't see any reason to not sync the fork to the main project.

Reply:
I have created a PR #17 to update our master, please approve. After that, your changes should be obsolete.